Paintroid/src/main/java/org/catrobat/paintroid/common/Constants.java

```diff
@@ -47,6 +47,7 @@
 	public static final String ORA_INFORMATION_DIALOG_TAG = "orainformationdialogfragment";
 	public static final String CATROBAT_INFORMATION_DIALOG_TAG = "catrobatinformationdialogfragment";
 	public static final String CATROID_MEDIA_GALLERY_FRAGMENT_TAG = "catroidmediagalleryfragment";
+	public static final String INDETERMINATE_PROGRESS_DIALOG_TAG = "progressdialogtag";
```

Comment:
```suggestion
	public static final String INDETERMINATE_PROGRESS_DIALOG_TAG = "indeterminateprogressdialogfragment";
```

Reply:
what do you think about writing it like this? (just for uniformity with other tags)

Reply:
yes 👍 good idea